src/main/java/uk/gov/hmcts/reform/preapi/services/BookingService.java

```diff
@@ -169,9 +169,11 @@ public void markAsDeleted(UUID id) {
             throw new NotFoundException("Booking: " + id);
         }
         var booking = entity.get();
+        booking.setDeleteOperation(true);
+        booking.setDeletedAt(Timestamp.from(Instant.now()));
         captureSessionService.deleteCascade(booking);
         shareBookingService.deleteCascade(booking);
```

Comment:
The cascade deletes will not be marked as deletes in the audits- this will happen for all cascade deletes in the other services as well (not just here)